yt_dlp/extractor/cbc.py

```diff
@@ -199,6 +238,10 @@ def _real_extract(self, url):
                     'force_smil_url': True
                 }),
             'id': video_id,
+            # needed because the m3u8 stream bandwidth info is wrong at least sometimes,
+            # making the format chooser prefer a supposedly higher quality file when it's just the same mp4 split up and served via hls.
+            # Since the bandwidth for downloading the direct mp4 does not seem to be limited, it's better to just download the whole mp4 file at once.
+            '_format_sort_fields': ('res', 'proto')
```

Comment:
```suggestion
            '_format_sort_fields': ('res', 'proto'),  # Prioritize direct http formats over HLS
```

Reply:
I think the info _why_ http is prioritized in this case is relevant, but I shortened down the comment by quite a bit. See if that fits the bill.